client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +144,49 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function useGetWpcomPlanTotalIfPaidMonthly( products: Array< ResponseCartProduct > ): {
+	[ key: string ]: number | undefined;
+} {
+	const plansToMonthly = products
+		.filter( ( product ) => {
+			return isWpComPlan( product?.product_slug ) && ! isMonthly( product?.product_slug );
+		} )
+		.map( ( product ) => {
+			return {
+				plan: product.product_slug,
+				monthly_plan: getMonthlyPlanByYearly( product.product_slug ),
+			};
+		} );
+	// Get unique non-empty monthly plan slugs
+	const monthlyPlanSlugs = [
+		...new Set( plansToMonthly.map( ( plan ) => plan.monthly_plan ).filter( ( plan ) => plan ) ),
+	] as PlanSlug[];
```

Comment:
This and above, it's a lot of looping over the same structure to get the non-empty array back 🤔

Reply:
Should be fixed.